### PR TITLE
[Azure AD] Added docs for authorization policy cmdlets

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by billmath** [comment]
Docs Build status updates of commit _[41f23ea](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/41f23ea0e75033f185a9f794953d6f0d143bb2c1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/authorization-policy-docs/azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/get-azureadmsauthorizationpolicy?view=azureadps-2.0&branch=pr-en-us-714) (azureadps-2.0) |
[azureadps-2.0/AzureAD/Set-AzureADMSAuthorizationPolicy.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/authorization-policy-docs/azureadps-2.0/AzureAD/Set-AzureADMSAuthorizationPolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/set-azureadmsauthorizationpolicy?view=azureadps-2.0&branch=pr-en-us-714) (azureadps-2.0) |
[mapping/groupMapping-2.0.json](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/authorization-policy-docs/mapping/groupMapping-2.0.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C14%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202109141225015679-714%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=xl4fMdb7GH71jHHUEkY9TtcR09aSTrMWn4E8AFVkJuY%3D&st=2021-09-14T12%3A21%3A05Z&se=2021-10-15T12%3A26%3A05Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
Fixes #713

**Comment by billmath** [comment]
Docs Build status updates of commit _[23ab688](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/23ab688bf36b3fdc93c42f46ac8f6d96bb894854)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/authorization-policy-docs/azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/get-azureadmsauthorizationpolicy?view=azureadps-2.0&branch=pr-en-us-714) (azureadps-2.0) |
[azureadps-2.0/AzureAD/Set-AzureADMSAuthorizationPolicy.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/authorization-policy-docs/azureadps-2.0/AzureAD/Set-AzureADMSAuthorizationPolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/set-azureadmsauthorizationpolicy?view=azureadps-2.0&branch=pr-en-us-714) (azureadps-2.0) |
[mapping/groupMapping-2.0.json](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/authorization-policy-docs/mapping/groupMapping-2.0.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C14%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202109141255024522-714%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=UGu9wILKNEUuoGwVUSPuv8WwjeZV1Yuhc4LTzu5C2UU%3D&st=2021-09-14T12%3A51%3A05Z&se=2021-10-15T12%3A56%3A05Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
@CelesteDG , please review this.

File: azureadps-2.0/AzureAD/Set-AzureADMSAuthorizationPolicy.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
```
Comment (CelesteDG) [inline azureadps-2.0/AzureAD/Set-AzureADMSAuthorizationPolicy.md:147]: @mswaweru Is it typical to have all these empty sections? If there's no content, can we remove them?

File: azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+This commands gets the Azure AD authorization policy.
+
+## PARAMETERS
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
```
Comment (CelesteDG) [inline azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md:36]: Sections with missing content. Is this typical? Can we remove the sections with missing info or no?

File: azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+external help file: Microsoft.Open.MS.GraphV10.PowerShell.dll-Help.xml
+Module Name: AzureAD
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureADMSAuthorizationPolicy
+
+## SYNOPSIS
+Gets an authorization policy.
```
Comment (CelesteDG) [inline azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md:11]: When I try to find a description for what an authorization policy is, I don't get any information/definition from our Azure AD IT docs. Instead, the first hit is in the Graph docs and some blogs that don't define what it is. Here's what a Graph resource topic says about it: "Represents a policy that can control Azure Active Directory authorization settings". 

Please update this description to say "Gets an authorization policy, which represents a policy that can control Azure Active Directory authorization settings." to give more context

**Review by CelesteDG — CHANGES_REQUESTED** [review]

**Comment by msewaweru** [comment]
Hi @CelesteDG, I have update the descriptions for both cmdlets. Please review.

**Pull request merged**